### ISSUE TITLE
There is no value for the neg_coeff

### ISSUE THREAD

**Issue by a-maumau** [dazhangyu123/OCL#2]
what value you used for the `neg_coeff`?

in `evaluate.py`
> loss = self.args.pos_coeff * pos_loss + self.args.neg_coeff * neg_loss

**Comment by dazhangyu123** [comment]
The default value of neg_coeff is 1.0, which can be located in the updated [evaluate.py](https://github.com/dazhangyu123/OCL/blob/main/evaluate.py) file

**Comment by a-maumau** [comment]
thank you for the update!
